client/Character.cpp

```diff
@@ -410,7 +425,7 @@ bool Character::loadAnimation(CharState state, string animFolder) {
 	return true;
 }
 
-bool Character::loadAnimationAssets(string assetFolder) {
+bool Character::loadAnimationAssets(string assetFolder, Direction d) {
```

Comment:
correct me if im wrong, but are you using the parameter Direction d? It seems like you won't need it, since you're getting direction from the index.txt